great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -1532,9 +1549,11 @@ def _chart_domain_values(
             alt.Chart,
         ]
         if plot_mode is PlotMode.PRESCRIPTIVE:
-            plot_impl = self.get_expect_domain_values_to_be_between_chart
+            if metric_name == "table_row_count":
```

Comment:
The fact that we have to do this probably means these will need to get pushed down into the child classes to support custom `DataAssistant`s